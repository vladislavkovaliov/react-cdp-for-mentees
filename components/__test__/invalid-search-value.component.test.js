import React from 'react';
import { InvalidSearchValue } from '../invalid-search-value.component';
import { shallow } from 'enzyme';

describe('<InvalidSearchValue />', () => {
  describe('Shapshot', () => {
    test('should render <InvalidSearchValue /> in invalid state', () => {
      const wrapper = shallow(
        <InvalidSearchValue isValid={false}>mocks children</InvalidSearchValue>
      );

      expect(wrapper).toMatchSnapshot();
    });

    test('should render <InvalidSearchValue /> in valid state', () => {
      const wrapper = shallow(
        <InvalidSearchValue isValid>mocks children</InvalidSearchValue>
      );

      expect(wrapper).toMatchSnapshot();
    });
  });
});