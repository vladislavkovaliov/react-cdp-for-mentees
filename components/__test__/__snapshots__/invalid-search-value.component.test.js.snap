// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InvalidSearchValue /> Shapshot should render <InvalidSearchValue /> in invalid state 1`] = `
<span
  className="header__content-input--invalid"
>
  mocks children
</span>
`;

exports[`<InvalidSearchValue /> Shapshot should render <InvalidSearchValue /> in valid state 1`] = `
<span
  className="hide"
>
  mocks children
</span>
`;
